proposals/NNNN-oracle-nio.md

```diff
@@ -0,0 +1,109 @@
+# Solution name
+
+* Proposal: [SSWG-NNNN](NNNN-oracle-nio.md)
+* Authors: [Timo Zacherl](https://github.com/lovetodream)
+* Review Manager: TBD
+* Status: **Awaiting implementation**
+* Implementation: [oracle-nio](https://github.com/lovetodream/oracle-nio)
+* Forum Threads: [Pitch](https://forums.swift.org/t/pitch-oraclenio-oracle-db-driver-built-on-swiftnio/69088)
+
+## Package Description
+A client library for Oracle databases based on swift-nio. It does neither require OCI nor ODPI.
+
+|  |  |
+|--|--|
+| **Package Name** | `oracle-nio` |
+| **Module Name** | `OracleNIO` |
+| **Proposed Maturity Level** | [Sandbox](https://github.com/swift-server/sswg/blob/main/process/incubation.md#process-diagram) |
```

Comment:
I think this is aligned with [Sandbox minimum requirements](https://www.swift.org/sswg/incubation-process.html#minimal-requirements). 
@FranzBusch anything missing ?

Reply:
Seems like it ticks all the boxes for Sandox